index.html

```diff
@@ -223,6 +223,52 @@ <h3>
           </ul>
         </div>
       </section>
+      <section>
+        <p class="contact-paragraph">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+        <form
+          id="contact-form"
+          action="https://formspree.io/f/xknlnqre"
+          method="post"
+        >
+          <input
+            type="text"
+            name="name"
+            maxlength="30"
+            placeholder="Full name"
+          />
+          <input type="email" name="email" placeholder="Email address" />
+          <textarea
+            name="message"
+            id="message"
+            type="text"
+            maxlength="500"
+            placeholder="Write me something..."
+          ></textarea>
+          <button class="formbutton" type="submit">Get in touch</button>
```

Comment:
> Implement the button interactions (enable, hover, etc.).

- [ ] The button interactions like the hover is not implemented in this submit button as stated in the project requirement. I'll suggest you look at the Figma designs dn get the text and background color for different button states. :wave: 

- - FIGMA:
![Screenshot from 2023-07-11 13-50-12](https://github.com/chiedozie013/portfolio-microverse/assets/107632780/d01f64d5-f311-452d-ba84-6d51ed13585f)

- [ ] Also I'll suggest you adjust the height of this button to fully match the Figma design as your button height is a bit smaller than the given design: :+1: :+1: